src/main/java/Duke.java

```diff
@@ -29,26 +28,26 @@ public static void main(String[] args) {
                     }
                     System.out.println(lists[i]);
                 }
-                System.out.println(divider);
+                System.out.println(DIVIDER);
                 line = in.nextLine();
             }  else if (line.contains("done")) {
                 int taskDone = Integer.parseInt(line.substring(5));
-                System.out.println(divider);
+                System.out.println(DIVIDER);
```

Comment:
You can consider grouping all print function into the same place and all variables changed into the same place for max readability